napari/components/_layers_list/view/main.py

```diff
@@ -148,16 +152,17 @@ def dragMoveEvent(self, event):
         total = self.vbox_layout.count()//2 - 1
         insert = total - divider_index
         layer_name = event.mimeData().text()
-        index = self.layers.index(layer_name)
-        # If the widget being dragged hasn't moved above or below any other
-        # widgets then don't highlight any dividers
-        selected = (not (insert == index) and not (insert-1 == index))
-        # Set the selected state of all the dividers
-        for i in range(0, self.vbox_layout.count(), 2):
-            if i == 2*divider_index:
-                self.vbox_layout.itemAt(i).widget().setSelected(selected)
-            else:
-                self.vbox_layout.itemAt(i).widget().setSelected(False)
+        if layer_name in self.layers:
```

Comment:
this should be removed – we should always check if what's being dragged is actually a layer, in which case, its name should _always_ be in the layers list anyways

Reply:
done